src/main/java/tr/edu/ogu/ceng/service/CompanyService.java

```diff
@@ -24,19 +28,21 @@ public class CompanyService {
 	@Autowired
 	private CompanyRepository companyRepository;
 
+	@Autowired
+	private MessageResource messageResource;
+
 	public Page<CompanyDto> getAllCompanies(Pageable pageable) {
 		ModelMapper modelMapper = new ModelMapper();
 		Page<Company> companies = companyRepository.findAll(pageable);
 		Page<CompanyDto> companyDtos = companies.map(company -> modelMapper.map(company, CompanyDto.class));
-		log.info("Pageable companies fetched");
+		log.info(messageResource.getMessage("company.service.pageable.companies.fetched"));
```

Comment:
loglar için dil değişikliği elzem bir durum değil, son kullanıcıya giden mesajların dil olarak ayrıştırılması gerekli.